demos/motmetrics4norfair/motmetrics4norfair.py

```diff
@@ -20,25 +22,29 @@
     help="Path to the MOT Challenge train dataset folder (test dataset doesn't provide labels)",
 )
 parser.add_argument(
-    "--make_video",
+    "--make-video",
+    dest="make_video",
```

Comment:
I don't think its necessary to add `dest="make_video"`, this library automatically converts `-` to `_` in the argument string.